docs/content/getting-started/releases.mdx

```diff
@@ -1,28 +1,41 @@
-- An exception to the above is “experimental APIs”, which allow us to offer new APIs to users and rapidly iterate based on their feedback. Experimental APIs will emit a warning whenever used, and may be broken even in dot releases.
-- As often as possible, deprecation warnings will precede removals. E.g. if we’re currently on version 0.8.5, and we wish to remove an API, we’ll start issuing a deprecation warning when the API is used, and then we’ll omit it from 0.9.0.
+### Experimental APIs
+
+The "experimental" marker allows us to offer new APIs to users and rapidly iterate based on their feedback. Experimental APIs are marked as such in the [API reference](/\_apidocs) and usually raise an <PyObject object="ExperimentalWarning" /> when used.
+
+Experimental APIs may change or disappear within any release, but we try to give warning for experimental APIs that have been around for a long time.
+
+### Deprecated APIs
+
+The "deprecated" marker indicates that we recommend avoiding an API, usually because there's a preferred option that should be used instead.
+
+Like non-deprecated public stable APIs, deprecated public stable APIs will not break within any major release after 1.0.
+
+## Dagster integration libraries
+
+Dagster's integration libraries are still in the 0.y.z of [SemVer](https://semver.org/). While technically this is "anything goes", we are conservative about making changes and provide guidance about when to expect breaking changes:
+
+- Upgrading to a new dot version within a minor release, e.g. 0.8.1 → 0.8.2, should never result in broken code.
+- An exception to the above is "experimental" APIs, as [described above](#experimental-apis).
+- As often as possible, deprecation warnings will precede removals. E.g. if we’re currently on version 0.8.5, and we wish to remove an API, we’ll start issuing a deprecation warning when the API is used, and then we'll omit it from 0.9.0.
 - Upgrading to a new minor version, e.g. 0.7.5 → 0.8.0 may result in breakages or new deprecation warnings.
-- We'll aim to batch deprecations every four weeks (or so).
 
 ## Following Along
 
-The best way to stay on top of what changes are included in each release is through the [changelog](https://docs.dagster.io/changelog). We call out breaking changes in the “Breaking Changes” section of the changelog and deprecations in the “Deprecations” section.
-
-We also publish a [migration guide](https://docs.dagster.io/migration) with each minor release, which details the changes you need to make to upgrade from the last minor release.
+The best way to stay on top of what changes are included in each release is through the [changelog](https://github.com/dagster-io/dagster/blob/master/CHANGES.md). We call out breaking changes in “Breaking Changes” sections and deprecations in “Deprecations” sections.
```

Comment:
Should the changelog link go to https://docs.dagster.io/changelog?

Reply:
👍